.github/workflows/sandbox-creating.yml

```
@@ -147,12 +147,26 @@ jobs:
             exit 1
           fi
 
+      - name: Check GitHub Token Access
+        run: |
+          RESPONSE=$(curl -s -H "Authorization: Bearer ${{ secrets.PAT_WITH_REPO_PERMISSIONS }}" \
+            -H "Accept: application/vnd.github+json" \
+            "https://api.github.com/repos/YOUR_ORG/YOUR_REPO/pulls")
+        
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)